internal/services/engines/csharp/rules.go

```diff
@@ -28,7 +28,7 @@ import (
 func NewCommandInjection() text.TextRule {
 	return text.TextRule{
 		Metadata: engine.Metadata{
-			ID:          "782ad071-1cf3-4230-936f-b7a1e794828d",
+			ID:          "HS-CSHARP-1",
```

Comment:
```suggestion
C#
```

Reply:
@matheusalcantarazup I think this is way is cleaner.
What do you think?

Reply:
I personally prefer CSHARP. What do you guys thing? @wiliansilvazup @nathanmartinszup?

Reply:
Throughout all the Horusec we call it `CSHARP`, maybe it's better keping equal

Reply:
When horusec show your analysis the language is `Csharp`. So i think it's better to keep `CSHARP` in this case.